docs/mobile.md

```diff
@@ -48,24 +52,25 @@ These instructions will allow you to debug your phone browser with Chrome DevToo
 
 1. In your Android phone, turn on USB debugging. This will allow your development computer to connect to your phone.
 
-   **USB debugging** is a setting the **Developer options** menu. This menu may be hidden on your phone. It can be revealed with a ["magic tap"](https://developer.android.com/studio/debug/dev-options) on the phone **Build number** 5 times.
-
+   **USB debugging** is a setting the **Developer options** menu. This menu may be hidden on your phone. It can be revealed with a ["magic tap"](https://developer.android.com/studio/debug/dev-options) on the phone **Build number** 7 times.
```

Comment:
can we change this to `is a setting in the...`

Reply:
Yes!  Thanks for catching that.